internal/gitstream/sync.go

```diff
@@ -177,6 +183,17 @@ func (s *Sync) Run(ctx context.Context) error {
 	return nil
 }
 
+func (s *Sync) getAuthorsToSkip() map[string]struct{} {
```

Comment:
I would rename this to `makeSet`.

Reply:
`makeSet` isn't descriptive IMO. How about `getIgnoredAuthorsSet`?

Reply:
Changed to `getIgnoreAuthorsSet`